### ISSUE TITLE
Automapper IMapper stops Fill with Mocks 

### ISSUE THREAD

**Issue by andychin7777** [Abc-Arbitrage/Abc.MoqComplete#19]
Sorry if this is a noob issue, 

With the new automapper IMapper needs to be injected into the constructors

MoqComplete is not able to show fill with mocks, 
![image](https://user-images.githubusercontent.com/74134581/106855289-3912de00-6704-11eb-8ffa-8234ddce8b85.png)

Removing the IMapper allows it to work though.
![image](https://user-images.githubusercontent.com/74134581/106854852-8fcbe800-6703-11eb-9d7c-4e3775ce2505.png)

Is there a way for moqComplete to work correctly with third party library interfaces?  Or will it not be able to see them?

**Comment by lconstan** [comment]
I'm going to investigate this.
Can you please share the signature of IMapper, IUserDal, and also your test class? I wonder where/how all these are defined and declared (in particular, are they nested?)

**Comment by andychin7777** [comment]
I've created simple test and dummy interface to show it better than trying to dump my work code in, the problem still occurs, but works fine if IMapper is commented out.  (Note:  I needed to restart Visual studio to get the project scanned again for resharper plugin to work.)

![image](https://user-images.githubusercontent.com/74134581/106866000-76329c80-6713-11eb-8d39-82ef820a582f.png)


```
[TestFixture]
public class UnitTest1
{
    [Test]
    public void TestMethod1()
    {
        var moq = new Mock<TestClass>();
        var testClass = new TestClass();
    }
}
```
```
public class TestClass
{
    private IMapper _thisMapper;
    private IDummyInterface _dummy;
    public TestClass(IMapper thisMapper, 
        IDummyInterface dummy)
    {
        _thisMapper = thisMapper;
        _dummy = dummy;
    }
}

public interface IDummyInterface
{
    void DummyFunction(bool isDummy);
}
```

**Comment by lconstan** [comment]
I have reproduced your issue (with IMapper coming from external library).
Thanks for pointing this out!

I'm going to fix it

**Comment by lconstan** [comment]
The issue should be fixed with version 1.4.4; should be available soon.
Thanks!

**Comment by lconstan** [comment]
Closing as the last version should be available. Please reopen if the bug remains
